source/features/code-highlight.js

```diff
@@ -0,0 +1,70 @@
+import {h} from 'dom-chef';
+import prism from 'prismjs';
+import 'prismjs/components/prism-jsx';
+import 'prismjs/components/prism-bash';
+import 'prismjs/components/prism-git';
+import 'prismjs/components/prism-typescript';
+import 'prismjs/components/prism-scss';
+
+const supportedLang = {
+	javascript: new Set(['js']),
+	jsx: new Set(['react', 'reactjsx']),
```

Comment:
I have never seen `react` or `reactjsx` used before. I don't think anyone would use this, so I would remove it.